src/views/InteractiveMap/Card.vue

```diff
@@ -36,6 +36,14 @@
       </v-btn>
     </v-toolbar>
     <v-card-title primary-title class="py-2" v-if="isSelected">
+      <v-alert
+        :value="card.solverStatus && card.solverStatus !== 'optimal'"
```

Comment:
Could this use `hasSimulationError`?

Reply:
Not really, that's used for handling HTTP-level errors like network issues or unhandled backend exceptions.